modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/model/listener/SystemObjectDefinitionMetadataModelListener.java

```diff
@@ -163,6 +166,10 @@ private void _executeObjectActions(
 		}
 	}
 
+	private String _getClassName(long companyId, String className) {
+		return className + StringPool.POUND + companyId;
+	}
```

Comment:
Hi @carolmariaabb 

I found a better approach to solve the problem. I will make force push with the new approach right now. Ok?

Reply:
@murilostodolni what approach? I suggested this change:
```
private String _getClassName(T baseModel) {
      return _modelClass.getName() + StringPool.POUND + _getCompanyId(baseModel);
}
```
 to avoid duplicate code because of `_modelClass.getName()` (called 3 times) and `_getCompanyId(baseModel)` (called 3 times)

Reply:
My bad, you are right. I was speaking about other approach of next [commit](9995d5960f1310aa30e4a8c7f595f4172ca1c308), sorry.